prompt_templates/zero_shot/zero_shot_prompt.py

```diff
@@ -0,0 +1,72 @@
+from pydantic import create_model
+import re
+
+
+class ZeroShotPrompt:
```

Comment:
I imagine there are some LangChain inspirations here, which would be ok. But I am wondering if it makes sense to actually use LangChain in defining the prompts. For instance, to inherit from one of their definitions. I genuinely don't know if it's worth doing it. So I am just asking a question here.